### PR TITLE
Improve default files and point to documentation

### PR DESCRIPTION
**Pull request by roesslerj**

### CODE REVIEW

File: src/main/resources/default-recheck.ignore [hunk 1]
```diff
@@ -1 +1,13 @@
...
+# matcher: retestid=div-b4f23, attribute: font.*
+
+# To ignore whole subtrees, use e.g. xpath:
+# matcher: xpath=HTML[1]/BODY[1]/DIV[1]/DIV[1]/DIV[1]
+
+# To ignore attributes globally, use 
+# attribute=class
+
+# More details and examples can be found here:
+# https://github.com/retest/docs/blob/master/docs/recheck/how-ignore-works-in-recheck.md
```
Comment (beatngu13) [inline src/main/resources/default-recheck.ignore]: I would suggest to always link against the deployed documentation: https://retest.github.io/docs/recheck/how-ignore-works-in-recheck/.

File: src/main/resources/default-recheck.ignore.js [hunk 2]
```diff
@@ -1 +1,9 @@
-// Ignore for recheck. Please do not delete, even if they appear to be empty
+// Here you can implement ignore rules for recheck in javascript.
+// Please do not delete this file, even if it is empty.
+
+// You can implement either of these two functions:
+// shouldIgnoreElement(element)
+// shouldIgnoreAttributeDifference(element, diff)
+
+// You can find more details and example rules at: 
+// https://github.com/retest/docs/blob/master/docs/recheck/how-ignore-works-in-recheck.md
```
Comment (beatngu13) [inline src/main/resources/default-recheck.ignore.js]: Here too: https://retest.github.io/docs/recheck/how-ignore-works-in-recheck/.

File: src/main/resources/default-recheck.ignore (hunk 1 above)
Comment (roesslerj) [reply src/main/resources/default-recheck.ignore]: Yes please, and it's also way shorter...

File: src/main/resources/default-recheck.ignore [hunk 3]
```diff
@@ -1 +1,13 @@
...
+# matcher: retestid=div-b4f23, attribute: font.*
+
+# To ignore whole subtrees, use e.g. xpath:
+# matcher: xpath=HTML[1]/BODY[1]/DIV[1]/DIV[1]/DIV[1]
+
+# To ignore attributes globally, use 
+# attribute=class
+
+# More details and examples can be found here:
+# https://retest.github.io/docs/recheck/how-ignore-works-in-recheck/
```
Comment (modulo11) [inline src/main/resources/default-recheck.ignore:13]: Maybe we should just link to the main page.

Comment (roesslerj) [reply src/main/resources/default-recheck.ignore:13]: In case we have a page for a specific topic ... why not link to it?

**Review by beatngu13 — APPROVED** [review]

**Pull request merged**